hedera-node/src/main/java/com/hedera/services/txns/schedule/ScheduleCreateTransitionLogic.java

```diff
@@ -82,6 +83,10 @@ public ResponseCodeEnum validate(TransactionBody txnBody) {
             return validity;
         }
 
+        if (op.getMemo().size() > 100) {
```

Comment:
`100` must be a global dynamic property

Reply:
check if present. otherwise will throw NPE